src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -149,31 +152,33 @@ public RowIdMatch mapValuesToRows(bio.terra.metadata.Snapshot snapshot,
         String datasetTableName = column.getTable().getName();
         String datasetBqDatasetName = prefixName(source.getDataset().getName());
 
-        StringBuilder builder = new StringBuilder();
```

Comment:
Should we filter soft deleted rows here? Someone is might be handing in values of rows that have been soft deleted. Not sure we should show them.

Maybe just leave it, since when we revise the UI for query builder, this method probably goes away.